recipe/meta.yaml

```diff
@@ -64,3 +66,4 @@ extra:
     - pitrou
     - jcrist
     - martindurant
+    - ogrisel
```

Comment:
@ogrisel do you consent to being listed as a maintainer for the `distributed` package on conda-forge?